app/Http/Controllers/JobController.php

```diff
@@ -135,6 +135,52 @@ public function create(Request $request) {
         //No job details exist yet because we're creating a new one
         $job = [];
 
+        $skillLangs = Lang::get('common/skills');
+
+        $softSkills = Skill::whereHas('skill_type', function ($query) {
+            $query->where('name', '=', 'soft');
+        })->get()
+            ->mapWithKeys(function ($skill) use ($skillLangs) {
+                return [
+                    $skill->id => $skillLangs['skills'][$skill->name]['name']
+                ];
+            })
+            ->all();
+
+        $hardSkills = Skill::whereHas('skill_type', function ($query) {
+            $query->where('name', '=', 'hard');
+        })->get()
+            ->mapWithKeys(function ($skill) use ($skillLangs) {
+                return [
+                    $skill->id => $skillLangs['skills'][$skill->name]['name']
+                ];
+            })
+            ->all();
+
+        asort($softSkills, SORT_LOCALE_STRING);
+        asort($hardSkills, SORT_LOCALE_STRING);
```

Comment:
Having issues here getting the French translations to sort properly. Any skills that start with an accented character are ending up at the bottom of the list. The locale seems to be set properly, might just be a local environment issue.

https://stackoverflow.com/a/2898539